### PR TITLE
[CI] Better hyperparameters for Pyramids-SAC, WalkerStatic-SAC, and Reacher-PPO

### PR DESCRIPTION
**Pull request by ervteng**
### Proposed change(s)

Better hyperparameters for the 3 environments above. Should train more reliably (especially Pyramids-SAC) and have a higher final reward. Requires https://github.com/Unity-Technologies/ml-agents-cloud/pull/150 to pass daily tests.

### Types of change(s)

- [x] Bug fix
- [ ] New feature
- [ ] Code refactor
- [ ] Breaking change
- [ ] Documentation update
- [ ] Other (please describe)

### Checklist
- [ ] Added tests that prove my fix is effective or that my feature works
- [ ] Updated the [changelog](https://github.com/Unity-Technologies/ml-agents/blob/master/com.unity.ml-agents/CHANGELOG.md) (if applicable)
- [ ] Updated the [documentation](https://github.com/Unity-Technologies/ml-agents/tree/master/docs) (if applicable)
- [ ] Updated the [migration guide](https://github.com/Unity-Technologies/ml-agents/blob/master/docs/Migrating.md) (if applicable)

### CODE REVIEW

File: config/sac/Pyramids.yaml [hunk 1]
```diff
@@ -5,32 +5,32 @@ behaviors:
...
       tau: 0.01
       steps_per_update: 10.0
       save_replay_buffer: false
       init_entcoef: 0.01
       reward_signal_steps_per_update: 10.0
     network_settings:
       normalize: false
-      hidden_units: 256
-      num_layers: 2
+      hidden_units: 512
```
Comment (andrewcoh) [inline config/sac/Pyramids.yaml:17]: Did you run with just the larger buffer?

Comment (ervteng) [reply config/sac/Pyramids.yaml:17]: Yes, it does get a better reward but it peaks out much earlier (around 1.2-ish) and the losses continue to climb.
